monai/bundle/reference_resolver.py

```diff
@@ -121,8 +123,14 @@ def _resolve_one_item(self, id: str, waiting_list: Optional[Set[str]] = None, **
             waiting_list = set()
         waiting_list.add(id)
 
-        ref_ids = self.find_refs_in_config(config=item_config, id=id)
-        for d in ref_ids:
+        for t, v in self.items.items():
```

Comment:
Is this a workaround to ensure `imports` expression are before real code?
Thanks for the update, let me think about it and get back to you soon.

Thanks.